docs/studio/groups/group-rules.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: "Group Rules"
+description: "Configure the resources accessible to your groups."
+icon: "database"
+---
+
+A **group rule** defines the roles and associated resources that determine what group members and API keys can access, as well as the level of permissions granted.
+
+<Frame caption="Group with no rules">
+    <img src="/images/studio/group-rules.png" />
+</Frame>
+
+When a group rule doesn't have any explicit resources, the group will always have access to all resources within the organization.
```

Comment:
I don't see much value in demonstrating a group without rules. This can be mentioned in a separate sentence.

> When a group rule doesn’t have any explicit resources, the group will always have access to all resources within the organization.

This is not clear. Someone could also interpret that an group without rules has access to all permissions.